src/Dfe.Spi.GraphQlApi.Domain/Dfe.Spi.GraphQlApi.Domain.csproj

```diff
@@ -5,6 +5,7 @@
     </PropertyGroup>
 
     <ItemGroup>
+      <PackageReference Include="Dfe.Spi.Models" Version="1.0.0" />
```

Comment:
Not sure if this is the latest version... might be worth updating, although we're going to have to soon anyway to include the extra fields for the demo.